frontend/views/containers/dashboard/GroupMembersActivity.vue

```diff
@@ -86,20 +86,20 @@ export default ({
   computed: {
     ...mapGetters([
       'groupStreaks',
-      'userDisplayName',
+      'userDisplayNameFromID',
       'groupProfiles'
     ]),
     onTimePayments () {
       return Object.entries(this.groupStreaks.onTimePayments || {})
         .filter(([username, streak]) => streak >= STREAK_ON_TIME_PAYMENTS)
         .sort((a, b) => b[1] - a[1])
-        .map(([username, streak]) => L('{user} - {count} month streak', { user: this.userDisplayName(username), count: streak }))
+        .map(([username, streak]) => L('{user} - {count} month streak', { user: this.userDisplayNameFromID(username), count: streak }))
     },
     missedPayments () {
       return Object.entries(this.groupStreaks.missedPayments || {})
         .filter(([username, streak]) => streak >= STREAK_MISSED_PAYMENTS)
         .map(([username, streak]) => {
-          const Largs = { user: this.userDisplayName(username), streak }
+          const Largs = { user: this.userDisplayNameFromID(username), streak }
```

Comment:
`userDisplayNameFromID` sounds like it takes an ID but here it's taking a username.. can fix?